jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -162,6 +163,65 @@ def create(
     )
 
 
+@app.command()
+@handle_abort
+def render_locally(
+    ctx: typer.Context,
+    application_path: pathlib.Path = typer.Option(
+        ...,
+        help="The path to the application directory to use as a template for the job script.",
+    ),
+    job_script_name: str = typer.Option(
+        ...,
+        help="The name of the job script to render locally.",
+    ),
+    output_path: pathlib.Path = typer.Option(
+        ...,
+        help="The path to the directory where the rendered job script should be saved.",
+    ),
```

Comment:
I wonder if we can reduce the typing to run it. I think we could make `application_path` an argument since it is the more important here, so it is positional and users do not need to type `--application_path`. Use would be like `jobbergate render-job-script-locally ../../examples/simple-application`.

We could also default both paths to the current working directory and have some default name for extra reduction on typing.

What do you think?

```suggested
@app.command()
@handle_abort
def render_locally(
    ctx: typer.Context,
    application_path: pathlib.Path = typer.Argument(
        pathlib.Path("."),
        help="The path to the application directory to use as a template for the job script.",
        dir_okay=True,
    ),
    job_script_name: str = typer.Option(
        "rendered-locally",
        help="The name of the job script to render locally.",
    ),
    output_path: pathlib.Path = typer.Option(
        pathlib.Path("."),
        help="The path to the directory where the rendered job script should be saved.",
    ),
```